alacritty_terminal/src/tty/unix.rs

```diff
@@ -328,7 +329,7 @@ impl EventedPty for Pty {
     #[inline]
     fn next_child_event(&mut self) -> Option<ChildEvent> {
         self.signals.pending().next().and_then(|signal| {
-            if signal != sighook::SIGCHLD {
+            if signal != sighook::consts::SIGCHLD {
```

Comment:
Might be overly pedantic, but having two module import depth here with the root module being a rename of an import seems unnecessarily complicated.

I think I'd prefer `consts::SIGCHLD`, just `SIGCHLD` (assuming we aren't using any signals from libc here), or an alias on the lower level like `sigconsts::SIGCHLD`. I'd say all of these are superior to the current solution and would be acceptable.

Reply:
There are uses of libc signals, but they're all namespaced (`libc::SIG...`).  I think `sigconsts` aligns nicely with the existing alias of `sighook`.